test/e2e/kubernetes/pvc/pvc.go

```diff
@@ -83,6 +99,16 @@ func Create(pvcName string, storageRequest string) (*Builder,error) {
 	return pvcBuilder,nil
 }
 
+func (b *Builder)WithAnnotations(annotations map[string]string) *Builder{
+	b.pvc.Annotations = annotations
+	return b
+}
+
+func (b *Builder)WithStorageClass(scName string) *Builder{
```

Comment:
```suggestion
func (b *Builder) WithStorageClass(scName string) *Builder{
```
Nit: Same for `DeployPVC` as well

Reply:
This calls for adding linters for test files too. One by one -- in a next PR. 
But will fix the lint issue manually for this PR as of now. 
Thanks @Jont828